server/container_create_linux.go

```diff
@@ -126,6 +126,10 @@ func (s *Server) finalizeUserMapping(specgen *generate.Generator, mappings *idto
 		return
 	}
 
+	if sb.Annotations()[crioann.UsernsModeAnnotation] == "" {
```

Comment:
Should we check for a unset field here?

```suggestion
	if _, ok := sb.Annotations()[crioann.UsernsModeAnnotation]; !ok {
```